tests/test_sets/learn_data_quality_use_cases/volume_financial_transfers.csv

```diff
@@ -0,0 +1,39 @@
+type,sender_account_number,recipient_fullname,transfer_amount,transfer_date
+domestic,244084670977,Jaxson Duke,9143.40,2024-05-01 01:12
+domestic,954005011218,Nelson O’Connell,3285.21,2024-05-01 05:08
+domestic,842374923847,Johanna McKee,5783.18,2024-04-12 15:35
+domestic,673894027340,Albert Rakes,8493.14,2024-04-21 09:50
+domestic,129384756019,Eliza Thornton,2756.89,2024-05-01 14:23
+domestic,587234901267,Oscar Wilde,6234.56,2024-05-01 18:45
```

Comment:
@Quantisan One request before I move this article into internal review - could you please use synthetic names for the sample transfer recipients, instead of historical figures? Thanks!